fusionauth/resource_fusionauth_tenant.go

```diff
@@ -30,7 +30,6 @@ func newTenant() *schema.Resource {
 				Type:       schema.TypeList,
 				MaxItems:   1,
 				Optional:   true,
-				Computed:   true,
```

Comment:
Sorry, bounced between the config mode and computed values here...

The server won't return any computed values in this block, so by marking this as 'not computed' it stops Terraform from expecting a value to be returned from the server if `access_control_configuration` is not set/configured.